encryption_works.md

```diff
@@ -1,5 +1,29 @@
 # Encryption Works: A Guide to Protecting Your Privacy for Journalists, Sources, and Everyone Else
 
+## August 2015
+
+## Written by Micah Lee. Edited by Tommy Collison, Harlo Holmes, and Garrett Robinson.
+
+*Dedicated to cypherpunks, and to whistleblowers past and future.*
+
+A publication of **Freedom of the Press Foundation**. Originally published July 2013, updated August 2015.
+
+Encryption Works: A Guide to Protecting Your Privacy for Journalists, Sources, and Everyone Else is licensed under a Creative Commons Attribution 3.0 Unported License. [https://creativecommons.org/licenses/by/3.0/](https://creativecommons.org/licenses/by/3.0/).
+
+If you're interested in contributing to Encryption Works, or have ideas for what this guide should cover, please submit issues on [GitHub](https://github.com/freedomofpress/encryption-works).
+
+<style>
+div#out-of-date {
+  background-color: #FFF263;
+  border: 1px solid #EFC933;
+  padding: 0px 16px;
+}
+</style>
+
+<div id="out-of-date">
+<p>Warning: This guide has not updated in over a year. Freedom of the Press Foundation is working on an updated version. If you're interested in contributing, or have ideas for what this guide should cover, please submit issues on <a href="https://github.com/freedomofpress/encryption-works">GitHub</a>
+</div>
```

Comment:
This PR re-adds the "out-of-date" warning that was just removed in #90. This looks like an incorrect resolution of the merge conflict you were having earlier.